src/terminal.rs

```diff
@@ -86,14 +93,20 @@ impl Terminal {
     }
 }
 
+#[derive(Clone, Copy, Debug)]
+pub enum ImageFormat {
+    ITerm2,
+    Terminology,
+}
```

Comment:
Nice change :)

Can you add doc comments to the enum and each field, perhaps with a link to the documentation for each of these formats.  For iterm2 it's here: https://www.iterm2.com/documentation-images.html